doc/example/store.ipynb

```diff
@@ -218,25 +310,123 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Just as in CSV, writing the history to HDF5 takes a considerable amount of time. Note that HDF5 in combination with\n",
+    "`MultiProcessEngine()`will automatically create one file with filename `filename_{id}` for each start, put them in a\n",
+    "folder namen `filename` and create one HDF5 file with name `filename`, which combines all those.The `filename`-file is\n",
+    "intended to be also useable as a savefile for the end results. Note however that `filename` will **cease to function**\n",
+    "if you delete the folder."
```

Comment:
```suggestion
    "Just as in CSV, writing the history to HDF5 takes a considerable amount of time. Note that HDF5 history in combination with\n",
    "the parallelization engine `MultiProcessEngine()` will result in one file with filename `filename_{id}` for each start, which are stored in a\n",
    "folder named `filename`, then after optimization an HDF5 file named `filename` is created, which combines all start files. The combined file is\n",
    "intended to be also usable as a savefile for the end results. Note however that the combined file references the start files in the folder, hence the combined file will **cease to function**\n",
    "if you delete the folder of start files."
```

The way it's currently written, the following files would be generated if an output file named `results.hdf5` is specified:
- folder containing start files: `results.hdf5/`
- start files: `results.hdf5/results.hdf5_{n_start}`
- combined file: `results.hdf5`

Maybe instead illustrating with an example would make things easier to understand? e.g.:

If a user specifies a HDF5 output file named `my_results.hdf5` and uses a parallelization engine, then:
- a folder is created to contain partial results, named `my_results/` (the stem of the output filename)
- files are created to store the results of each start, named `my_results/my_results_{START_INDEX}.hdf5`
- a file is created to store the combined result from all starts, named `my_results.hdf5`. Note that this file depends on the files in the `results/` directory, so will become unusable if `my_results/` is deleted.